docs/source/guide/index.rst

```diff
@@ -938,6 +938,86 @@ This command allows downloading an artifact from a provided repository using the
     │ --help              -h        Show this message and exit.                                        │
     ╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
 
+.. rstuf-cli-artifact-repository
+
+Artifact Addition (``repository``)
+---------------------------
+
+This command provides artifact repository management for the RSTUF repository config.
+
+.. code::
+
+    ❯ rstuf artifact repository --help
+
+    Usage: rstuf artifact repository [OPTIONS] COMMAND [ARGS]...
+
+    Repository management.
+
+    ╭─ Options ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │ --help  -h    Show this message and exit.                                                                                                                                 │
+    ╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+    ╭─ Commands ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │ delete                           Delete repository.                                                                                                                       │
+    │ add                              Add a new repository.                                                                                                                    │
+    │ show                             List configured repositories.                                                                                                            │
+    │ update                           Update repository.                                                                                                                       │
+    │ set                              Switch current repository.                                                                                                               │
+    ╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+
+.. code::
+
+    ❯ rstuf artifact repository delete --help
+
+    Usage: rstuf artifact repository delete [OPTIONS] REPOSITORY
+
```

Comment:
Nit: add a line using the descriptions like `Delete a repository`.
Here is how it looks currently:
<img width="1118" alt="image" src="https://github.com/repository-service-tuf/repository-service-tuf-cli/assets/16246778/95c452a9-e62e-43e2-9b78-abf69d3bef21">